### ISSUE TITLE
Adding brackets brocken

### ISSUE THREAD

**Issue by pddzaic** [ruby/irb#373]
## Description

Inserting brackets is very difficult. Here an example while defining an array. When I first press the bracket-key, nothing happens, pressing it again results in:_

![Bild_001](https://user-images.githubusercontent.com/754440/175564344-1b951f99-c5df-4fa9-a040-b83e72f326e3.png)

## Result of irb_info

```
irb(main):001:0> irb_info
=> irb(main):001:0> irb_info
=>
Ruby version: 2.7.6
IRB version: irb 1.4.1 (2021-12-25)
InputMethod: ReidlineInputMethod with Reline 0.3.1
RUBY_PLATFORM: x64-mingw32
East Asian Ambiguous Width: 1
Code page: Aktive Codepage: 65001.
```

## Setting Files

I think not.

**Comment by Toymen** [comment]
### Same here 

![image](https://user-images.githubusercontent.com/51193141/199971740-47b648ca-2468-4d81-8b66-97694300a9a9.png)

**Comment by sfrieske** [comment]
This seems to be an issue of reline, not irb. I've created [reline issue #475](https://github.com/ruby/reline/issues/475) to address this. A crude workaround is described there.

**Comment by sfrieske** [comment]
Seems to be fixed in reline 0.3.2 in conjunction with IRB 1.6.2. I've closed the reline issue #475.

**Comment by hasumikin** [comment]
I'm closing this issue as I could not reproduce the problem on my Windows (cmd and PowerShell)
